src/OSPSuite.UI/Controls/UxHtmlLabel.cs

```diff
@@ -1,51 +1,40 @@
 using System;
 using System.Drawing;
+using System.Windows.Forms;
 using DevExpress.Utils.Controls;
 using DevExpress.XtraEditors.Controls;
 using DevExpress.XtraRichEdit;
+using AutoSizeMode = DevExpress.XtraRichEdit.AutoSizeMode;
 
 namespace OSPSuite.UI.Controls
 {
    public class UxHtmlLabel : UxRichEditControl, IXtraResizableControl
    {
-      private static readonly object _layoutInfoChanged = new object();
-      private static string _fontFamily;
```

Comment:
Not sure why all those guys were static. This looks like a smell to me (why would all Label share the same variable, especially the layoutInfoChanged??)